server/filters/formatDateFilter.ts

```diff
@@ -0,0 +1,6 @@
+import moment from 'moment'
```

Comment:
I have created a generic date formatting filter to replace the 2 that I have deleted below (this approach saved me creating a 3rd very specific filter)